cydaf/tomatoClockForPractice
# Report task/api

# why, what?
主要調整內容：
1. 後端些微程式碼
2. 前端 auth.js ->  register result 位置

# how to test
1. cd `./server`
2. `cp .env.copy .env`
3. TOKEN_SECRET= 
    USEREMAIL = 
    PASSWORD = 
    Port = 
    Redirect = 驗證後連結的網址
    verify = 驗證導入的 api/auth/verify 判斷使用者 token 是否正確
4. `nom run start`
